Gemfile

```diff
@@ -46,7 +46,7 @@ gem 'bootsnap', require: false
 # gem "sassc-rails"
 
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
-# gem "image_processing", "~> 1.2"
+# gem 'image_processing', '~> 1.2'
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
```

Comment:
- [ ] Once you have install the expected gem, then you will make sure the expected tests for this milestone are passing. Below, you can see the result of your tests:
![Screenshot from 2023-01-25 03-12-40](https://user-images.githubusercontent.com/83042742/214457657-e47c7208-1ae8-44e9-b1ea-e8061a91b4ef.png)
**For a short remind about testing your controllers, just refer to this **[article](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controller_test_lesson.md)****